### ISSUE TITLE
`BraceWrapping` doesn't seem to be working

### ISSUE THREAD

**Issue by tadeu** [zed0/clang-format-configurator#17]
For `clang-format 6.0.1`, I'm setting `BasedOnStyle` to `WebKit` and `BreakBeforeBraces` to `Custom`. When I change individually items in `BraceWrapping` they don't affect the code, and if I download the `clang-format` file, `BraceWrapping` is not there!

**Comment by SasquatchYuja** [comment]
Got the same issue (LLVM base style) with `BreakBeforeBraces: Custom` and options set in `BraceWrapping`. Output clang-format file contains `BreakBeforeBraces: Custom` but no change occurs whatever set of `BraceWrapping` I tick

**Comment by DominikDeak** [comment]
I can also confirm that this `BraceWrapping` is broken for `BreakBeforeBraces: Custom`. Even if I hand edit the `.clang-format` file and manually add `BraceWrapping`  flags, then re-upload, changes have no effect.

**Comment by KestrelAdrian** [comment]
Yep, I'm trying on safari and it's not working for me either.

**Comment by superKoder** [comment]
True. None of the checkboxes are actually written into the file either.
